pymc3/tests/test_distributions.py

```diff
@@ -695,6 +704,29 @@ def test_mvnormal_init_fail(self):
             with pytest.raises(ValueError):
                 x = MvNormal('x', mu=np.zeros(3), cov=np.eye(3), tau=np.eye(3), shape=3)
 
+    @pytest.mark.parametrize('n', [1, 2, 3])
+    def test_matrixnormal(self, n):
+        self.pymc3_matches_scipy(MatrixNormal, RealMatrix(5, n),
```

Comment:
Travis shows:

    self = <pymc3.tests.test_distributions.TestMatchesScipy object at 0x7f786acb5e80>
    n = 1
        @pytest.mark.parametrize('n', [1, 2, 3])
        def test_matrixnormal(self, n):
    >       self.pymc3_matches_scipy(MatrixNormal, RealMatrix(5, n),
                                     {'mu': Vector(R, n), 'rowcov': PdMatrix(n),
                                      'colcov': PdMatrix(n)},
                                     matrix_normal_logpdf_cov)
    E       NameError: name 'MatrixNormal' is not defined

Don't forget `MatrixNormal` import at the top!